ehr_billing/src/org/labkey/ehr_billing/EHR_BillingManager.java

```diff
@@ -1,18 +1,3 @@
-/*
```

Comment:
Looks like the copyright is disappearing. Can you restore?

Reply:
Restored!